tests/unit-tests/test-class-sensei-data-cleaner.php

```diff
@@ -432,4 +494,36 @@ public function testSenseiRolesAndCapsDeleted() {
 		$role = get_role( 'teacher' );
 		$this->assertNull( $role, 'Teacher role should be removed overall' );
 	}
+
+	/**
+	 * Ensure comments are trashed.
+	 *
+	 * @covers Sensei_Data_Cleaner::cleanup_all
+	 * @covers Sensei_Data_Cleaner::cleanup_comments
+	 */
+	public function testCleanupComments() {
```

Comment:
This test should go above the `/* Helper functions. */` section. I moved `testSenseiRolesAndCapsDeleted` in another PR.